README.md

```diff
@@ -353,11 +353,11 @@ Browse the following web site and download whichever built binary files you wish
 The build process builds some docker images with the command-line tools installed.
 This could be useful when wishing to embed a usage of the command-line within a build process which can use a docker image.
 
-- Bleeding edge/Unstable : `docker pull harbor.galasa.dev/galasadev/galasa-cli-amd64:main`
+- Bleeding edge/Unstable : `docker pull harbor.galasa.dev/galasadev/galasa-cli-x86_64:main`
```

Comment:
We are going to leave the Docker images with the same names for now so could you change this back please :)